client/dashboard/sites/backups/fields.tsx

```diff
@@ -0,0 +1,51 @@
+import { Icon } from '@wordpress/components';
+import { dateI18n } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+import { cloud, globe, image } from '@wordpress/icons';
+import type { ActivityLogEntry } from '../../data/types';
+import type { Field } from '@wordpress/dataviews';
+
+function getIcon( item: ActivityLogEntry ) {
+	switch ( item.name ) {
+		case 'attachment__uploaded':
+			return image;
+		case 'plugin__updated':
+			return globe;
+		default:
+			return cloud;
+	}
+}
+
+export function getFields(): Field< ActivityLogEntry >[] {
+	return [
+		{
+			id: 'date',
+			label: __( 'Date' ),
+			render: ( { item } ) => (
+				<>
+					<strong>{ dateI18n( 'F j, Y', item.published ) }</strong>
```

Comment:
I don't think `dateI18n()` handles localization correctly 🤔 can you try changing your wordpress.com/me/account language to non-English and see if it's translated?

I think we need to use existing `formatDate()` or `<TimeSince>` which handle it correctly (check the existing codebase on how we've been using that 😄)